src/plugins/accounting.cpp

```diff
@@ -0,0 +1,12 @@
+/************************************************************\
+ * Copyright 2024 Lawrence Livermore National Security, LLC
+ * (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+ *
+ * This file is part of the Flux resource manager framework.
+ * For details, see https://github.com/flux-framework.
+ *
+ * SPDX-License-Identifier: LGPL-3.0
+\************************************************************/
+
+#include "accounting.hpp"
```

Comment:
What is `accounting.cpp` for if all the code is in `accounting.hpp`? Do you plan to implement methods on the `Association` class in this file later?

Reply:
Yes, I was planning on adding any methods as well as standalone functions that relate to any `Accounting` objects in this file (I know of at least three at the moment).